src/main/java/service/models/User.java

```diff
@@ -22,8 +21,6 @@ public class User {
   private UserRoles role;
   private LocalDateTime lastAccess;
 
-  @Nullable private final UUID inventoryAccess;
```

Comment:
Same question as the inventory one. Are we keeping track who the inventory belongs to somewhere else? Im assuming in the handlers possibly?

Reply:
I dont think we should. Since anyone could access any inventory. There will be multiple users using multiple inventories. The junction table `User_inventories` will store every pair of user and inventory. Maybe we can add some logic to more routes specific to this junction table.